core/custom_filters/custom_filter_registry_point.py

```diff
@@ -0,0 +1,83 @@
+import logging
+import os
+
+from typing import List
+
+from core.custom_filters import CustomFilterWizardInterface
+
+
+logger = logging.getLogger(__name__)
+
+
+class CustomFilterRegistryPoint:
+    """
+    Class responsible for handling the registration of custom filters.
+
+    REGISTERED_CUSTOM_FILTER_WIZARDS:
+    A dictionary that collects registered implementations of custom filter wizards.
+    The structure of the dictionary is as follows:
+
+    {
+        'policy': [],
+        'social_protection': [
+            {
+                'class_reference': <class 'social_protection.custom_filters.BeneficiaryCustomFilterCreator'>,
+                'module': 'social_protection'
+            }
+        ]
+    }
+    """
+
+    REGISTERED_CUSTOM_FILTER_WIZARDS = {}
+
+    @classmethod
+    def register_custom_filters(
+        cls,
+        module_name: str,
+        custom_filter_class_list: List[CustomFilterWizardInterface]
+    ) -> None:
+        """
+            Register custom filter wizards for a specific module in the openIMIS application.
+
+            This method registers the provided list of objects as custom filters in the hub,
+            for the specified module in the openIMIS application.
+
+            Args:
+                module_name (str): The name of the module installed in openIMIS.
+                custom_filter_class_list (list): A list of objects representing the custom filter implementations.
+
+            Returns:
+                None: This method does not return anything.
```

Comment:
Correct me if I'm wrong but I think we're using reStructuredText format for dosctrings. If that's the case then please fix them

Reply:
Yes, I'm going to fix this to use reStructuredText